tests/test_meshtastic_utils_async.py

```
@@ -0,0 +1,662 @@
+        from mmrelay.meshtastic_utils import _fire_and_forget
+
+        with patch("mmrelay.meshtastic_utils._submit_coro") as mock_submit:
+            _fire_and_forget("not-a-coro")  # type: ignore[arg-type]
+
+        mock_submit.assert_not_called()
+
+    def test_fire_and_forget_returns_when_submit_none(self):
+        """Ensure fire-and-forget returns when _submit_coro yields no task."""
+        from mmrelay.meshtastic_utils import _fire_and_forget
+
+        async def _noop():
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_noop`

Add return type annotation: `None`

(ANN202)